feed/config.py

```diff
@@ -1,67 +1,30 @@
 """Flask configuration."""
 
 import os
-from feed import consts
-from zoneinfo import ZoneInfo
+from typing import List, Tuple
+
+import arxiv.config as arxiv_base
 
 
-class Config:
+from feed import consts
+
+class Settings(arxiv_base.Settings):
     """Base configuration object."""
 
     DEBUG = False
     TESTING = False
     VERSION = "1.1"
 
-    BASE_SERVER = os.environ.get("BASE_SERVER", "arxiv.org")
-    RSS_SERVER = os.environ.get("RSS_SERVER", "rss.arxiv.org")
-    INFO_SERVER = os.environ.get("INFO_SERVER", "info.arxiv.org")
     STATIC_SERVER = os.environ.get("STATIC_SERVER", "static.arxiv.org")
 
     FEED_NUM_DAYS = os.environ.get("FEED_NUM_DAYS", consts.FEED_NUM_DAYS)
 
-    ARXIV_BUSINESS_TZ = ZoneInfo(os.environ.get('ARXIV_BUSINESS_TZ', 'America/New_York'))
-
-    URLS =[
-        ("pdf", "/pdf/<arxiv:paper_id>v<string:version>", BASE_SERVER),
-        ("pdf_by_id", "/pdf/<arxiv:paper_id>", BASE_SERVER),
-        ("taxonomy", "/category_taxonomy", BASE_SERVER),
-        ("help","/help/rss.html", INFO_SERVER),
+    ###add to the default URLS
+    URLS: List[Tuple[str, str, str]] = [
         ("static","/static/<path:file_path>", STATIC_SERVER)
     ]
 
-    ### database connection
-    SQLALCHEMY_DATABASE_URI = os.environ.get('SQLALCHEMY_DATABASE_URI', 'sqlite:///tests/data/test_data.db')
-    CLASSIC_DATABASE_URI = SQLALCHEMY_DATABASE_URI
+    ### database connection handled by base
```

Comment:
It may help people if you override in feed, since it's a pretty core value.
Base isn't going to know about your test_data.db file, so that might be a good default.
I see your readme note, which is good, but I would still lean toward having pytest work out of the box.